tendermint/tests/lite.rs

```diff
@@ -82,7 +84,7 @@ impl Requester<SignedHeader, Header> for MockRequester {
             return Ok(sh.into());
         }
         println!("couldn't get sh for: {}", &h);
-        Err(Error::RequestFailed)
+        fail!(Kind::RequestFailed, "couldn't get sh for: {}", &h);
```

Comment:
This is just a shortcut for 
```rust
return Kind::Kind::RequestFailed.context(format!("couldn't get sh for: {}", &h)).into();
```

Reply:
Okay, I was close but didn't know about this shortcut. Thanks!